src/node.cc

```diff
@@ -3261,6 +3258,9 @@ void LoadEnvironment(Environment* env) {
   // are not safe to ignore.
   try_catch.SetVerbose(false);
 
+  // Compile, execute the lib/internal/bootstrap_node.js file. (Which was
+  // included as static C string in node_natives.h.
```

Comment:
@cjihrig Thanks, I've added a commit based on your suggestions. Let me know what you think.